doc/man7/EVP_KDF-HKDF.pod

```diff
@@ -106,7 +106,7 @@ salt value "salt" and info value "label":
  unsigned char out[10];
  OSSL_PARAM params[5], *p = params;
 
- kdf = EVP_KDF_new("HKDF");
+ kdf = EVP_KDF_fetch("HKDF");
```

Comment:
The fetching functions take three parameters, where the middle one is the name. In these examples, you can leave the other two NULL

Reply:
I added fixup commits for these.  If you're fine with that, we're good to go.

Reply:
I'm fine with it, I fixed the two you missed :)

Reply:
Cool!